packages/millicast-sdk/tests/unit/LoggerDiagnose.steps.js

```diff
@@ -112,7 +112,6 @@ defineFeature(feature, (test) => {
       expect(viewer.webRTCPeer.getRTCPeerStatus()).toEqual('connected')
       viewer.webRTCPeer.initStats()
       const stats = await viewer.webRTCPeer.peer.getStats()
-      viewer.webRTCPeer.peerConnectionStats.parseStats(stats)
```

Comment:
question - not needed?

Reply:
my mistake, I deleted this line unintentionally